docbook/pom.xml

```diff
@@ -10,6 +10,11 @@
     <packaging>jdocbook</packaging>
     <name>RESTEasy Reference Guide (${translation})</name>
     <description/>
+    <properties>
+        <jbang.ver>0.45.0</jbang.ver>

```

Comment:
Only this minor comment that is not required for this to be merged IMO, we should just upgrade this to `0.67.3` as it's the current latest version.

Reply:
okay!